package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lob/ui-components",
-  "version": "2.0.0-beta.62",
+  "version": "2.0.0-beta.63",
```

Comment:
I think you need to update `package-lock.json` too

Reply:
Will do! Thank you